shub/log.py

```diff
@@ -1,11 +1,11 @@
-import click
-
 from shub.utils import get_job
 
+import click
+
 
 @click.command(help='Get log of a given job on Scrapy Cloud')
 @click.argument('job_id')
 def cli(job_id):
     job = get_job(job_id)
-    for item in job.logs.iter_values():
+    for item in job.logs.iter_json(format='text'):
```

Comment:
this will fail at first retried error at [hubstorage/resourcetype.py#L60](https://github.com/scrapinghub/python-hubstorage/blob/master/hubstorage/resourcetype.py#L60) called from  [hubstorage/resourcetype.py#L81](https://github.com/scrapinghub/python-hubstorage/blob/master/hubstorage/resourcetype.py#L81)